src/CaptureService/CaptureServiceImpl.cpp

```diff
@@ -370,7 +370,8 @@ grpc::Status CaptureServiceImpl::Capture(
 
   // Enable user space instrumentation.
   std::optional<std::string> error_enabling_user_space_instrumentation;
-  if (capture_options.enable_user_space_instrumentation()) {
+  if (capture_options.enable_user_space_instrumentation() &&
```

Comment:
I guess the same should be done below where `UninstrumentProcess` is called?

Reply:
In most cases that's not really changing anything. UninstrumentProcess immediatly returns if the process has not been instrumented before. Added anyway for the sake of having the same condition there.

Reply:
Ah, got it.